sympy/polys/factortools.py

```diff
@@ -1301,12 +1303,14 @@ def dmp_factor_list(f, u, K0):
             if K0_inexact is None:
                 coeff = coeff/denom
             else:
+                num_factors = 0
                 for i, (f, k) in enumerate(factors):
                     f = dmp_quo_ground(f, denom, u, K0)
```

Comment:
@jksuom That's diff mentioned in OP. It divides coeff by denom in `coeff = K0.quo(coeff, denom)` (I'm not sure about correct way to divide by denom). But it produces results like below
```
>>> factor(6.7225336055071*x**2 - 10.6463972754741*x - 0.33469524022264)
3.64897391178212e-40*(1.84230793862373e+40*x**2 - 2.91764137888136e+40*x - 9.17231112949169e+38)
```